js_modules/dagit/packages/core/src/runs/ScheduledRunsListRoot.new.tsx

```diff
@@ -35,9 +41,35 @@ export const ScheduledRunListRoot = () => {
 
   const refreshState = useQueryRefreshAtInterval(queryResult, FIFTEEN_SECONDS);
 
+  const runCountResult = useQuery<RunTabsCountQuery, RunTabsCountQueryVariables>(
+    RUN_TABS_COUNT_QUERY,
+    {
+      variables: {
+        queuedFilter: {statuses: Array.from(queuedStatuses)},
+        inProgressFilter: {statuses: Array.from(inProgressStatuses)},
+      },
+    },
+  );
+
+  const {data: countData} = runCountResult;
+  const {queuedCount, inProgressCount} = React.useMemo(() => {
+    return {
+      queuedCount:
+        countData?.queuedCount?.__typename === 'Runs' ? countData.queuedCount.count : null,
+      inProgressCount:
+        countData?.inProgressCount?.__typename === 'Runs' ? countData.inProgressCount.count : null,
+    };
+  }, [countData]);
+
```

Comment:
Hmm if we revisit this, it might be nice to break these ~15 lines out into a hook just to make sure any future changes to it are applied in both places

Reply:
Yeah good call, let me do that

Reply:
Ah I see, I need to convert the component into a hook because of the combined query refresh state call that happens within `RunRoot`